Students/Farcas Silviu Vlad/Assignemnt 1.1/Multiples.java

```diff
@@ -0,0 +1,11 @@
+public class HelloWorld {
+	
+	public static void main(String[] args) {
+		int i, sum = 0;
+		for (i = 1 ; i <= 1000 ; i++) {
```

Comment:
This is not really efficient. You are traversing 1000 numbers in order to get your result and also doing quite a few comparisons there!
Take a look [here](http://math.stackexchange.com/a/9305). You find a formula which gives you the result in just a few operations!
1000 may be small, however, consider a very large number like 1000000. How long would it take for your program to run? Would an `O(n)` complexity be worth it instead of simply applying a few formulas?